tools/ci_build/github/linux/run_dockerbuild.sh

```diff
@@ -48,6 +48,10 @@ elif [ $BUILD_OS = "centos7" ]; then
     IMAGE="centos7"
     DOCKER_FILE=Dockerfile.centos
     docker build -t "onnxruntime-$IMAGE" --build-arg BUILD_USER=onnxruntimedev --build-arg BUILD_UID=$(id -u) --build-arg PYTHON_VERSION=${PYTHON_VER} -f $DOCKER_FILE .
+elif [ $BUILD_OS = "centos6" ]; then
+    IMAGE="centos6gpu"
+    DOCKER_FILE=Dockerfile.centos6_gpu
```

Comment:
>Dockerfile.centos6_gpu [](start = 16, length = 22)

where is this file? check-in already?

Reply:
This is a different PR.

---
In reply to: [336321806](https://github.com/microsoft/onnxruntime/pull/2165#discussion_r336321806) [](ancestors = 336321806)